dags/databricks-ml-example.py

```diff
@@ -1,11 +1,10 @@
-from pendulum import datetime
 import logging
 
+import mlflow
```

Comment:
This packages is pretty large to attempt to import at parse time. Can you move this to the task that needs this package. The idea here is similar to avoiding what's outlined in the [Airflow best practice doc](https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#top-level-python-code).

Reply:
yup, thanks for the call out, meant to change this after our earlier discussion :)